mmv1/templates/terraform/resource_iam.html.markdown.erb

```diff
@@ -217,6 +217,9 @@ resource "<%= resource_ns_iam -%>_member" "member" {
 
 The following arguments are supported:
 
+<% object.root_properties.select(&:required).each do |prop| -%>
```

Comment:
Note that this seems to generate bad entries for `name` and `location` in some cases

Reply:
And invalid ones like `replica_zones` for region disk

Reply:
Noted, for some iam files this looks like generated duplicated entries for `name` and `location`